core/src/main/java/jenkins/security/s2m/DefaultFilePathFilter.java

```diff
@@ -59,6 +62,15 @@ protected boolean op(String op, File f) throws SecurityException {
                     LOGGER.log(Level.FINE, "agent allowed to {0} {1}", new Object[] {op, f});
                     return true;
                 } else {
+                    try {
+                        Field current = Request.class.getDeclaredField("CURRENT");
+                        current.setAccessible(true);
+                        Field createdAt = Command.class.getDeclaredField("createdAt");
+                        createdAt.setAccessible(true);
```

Comment:
Good enough for now; could be cleaned up by introducing a proper API in `remoting`. I am not inclined to bother if #5885 is under consideration anyway.